doc/languages-frameworks/javascript.section.md

```diff
@@ -871,6 +871,251 @@ stdenv.mkDerivation (finalAttrs: {
 })
 ```
 
+### buildDenoPackage {#javascript-buildDenoPackage}
+
+`buildDenoPackage` allows you to package Deno-based projects in Nixpkgs without the use of an auto-generated dependencies file (as used in [node2nix](#javascript-node2nix)).
```

Comment:
```suggestion
`buildDenoPackage` allows you to package [Deno](https://deno.com/) projects in Nixpkgs without the use of an auto-generated dependencies file (as used in [node2nix](#javascript-node2nix)).
```

It would help readers unfamiliar with a particular corner of the IT ecosystem if random-looking strings like project names always had a link to their definition. Especially if they're short, because in that case web search can be completely useless.